utils/python/CIME/SystemTests/system_tests_common.py

```diff
@@ -199,6 +199,10 @@ def run_indv(self, suffix="base", st_archive=False):
             infostr += ", with restarts every %d %s"%(rest_n, rest_option)
         logger.info(infostr)
 
+
```

Comment:
There was a request that test jobs waiting in the queue have a PEND status while test jobs currently running have a RUN status.   I think that this is the only place it's needed.